core/src/edu/asu/c3simulator/screens/DifficultySelectionScreen.java

```diff
@@ -185,5 +192,5 @@ public void show()
 	{
 		Gdx.input.setInputProcessor(stage);
 	}
-	
+
 }

```

Comment:
This was just the result of ctrl+shift+f. also, I linked it to mainhub/removed todo comment asking to link it to mainhub.